### DISCUSSION TITLE
Request: Pingvin-Share

### DISCUSSION THREAD

**Discussion by yodaphone** [tteck/Proxmox#1883] [Ideas]
It would be great if you can script this

https://github.com/stonith404/pingvin-share - A self-hosted file sharing platform. Preferably non docker version

**Comment by tteck** [comment]
If you would test and report your findings.
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/pingvin.sh)"
```

**Reply by yodaphone** [reply]
Works great. You may want to tag it as "Beta" as the application is in active development. 

Another quick one. How do updates work?

**Reply by tteck** [reply]
Run the installation script, or type `update` in the LXC Console
